lib/auto-languageclient.ts

```diff
@@ -638,7 +638,10 @@ export default class AutoLanguageClient {
       excludeLowerPriority: false,
       filterSuggestions: true,
       getSuggestions: this.getSuggestions.bind(this),
-      onDidInsertSuggestion: this.onDidInsertSuggestion.bind(this),
+      onDidInsertSuggestion: (event) => {
+        AutocompleteAdapter.applyAdditionalTextEdits(event)
```

Comment:
Why is this called here instead of in the `onDidInsertSuggestion` function?

Reply:
Because this is part of the LSP. The `onDidInsertSuggestion` is independent of the LSP. It is a method that people can override for custom behavior.